### DISCUSSION TITLE
Add DPGMedia website to `annoyances-cookies.txt`

### DISCUSSION THREAD

**Discussion by Nomes77** [uBlockOrigin/uAssets#18837] [Ideas] [answered]
Recently, in the `annoyances-cookies.txt` file a lot of website were added.
Would you mind adding also DPGMedia websites like `https://www.ad.nl/`, `https://www.ed.nl/`, `https://www.gelderlander.nl/`?

**Comment by krystian3w** [comment, marked as answer]
You should firstly exclude address problem by standard `set-constant` or newly global `set-cookie`/`set-local/session-storage-item` compatible with:
- EL Cookies, 
- AG Cookie Notices (these after filters will be not removed from used file with [`/ublock/`](https://filters.adtidy.org/extension/ublock/filters/18.txt) in catalog (directory) file name - for now file is without `set-cookie`/`set-local/session-storage-item` - 100+ exists in file https://filters.adtidy.org/extension/firefox/filters/18.txt) and
- at end with Netherland Lists with added section "close" hard cookie/GPDR modals.
